src/table.ts

```diff
@@ -778,133 +749,46 @@ Please use the format 'prezzy' or '${instance.name}/tables/prezzy'.`);
       rowStream?.removeListener('end', originalEnd);
     };
 
-    // eslint-disable-next-line @typescript-eslint/no-explicit-any
-    userStream.end = (chunk?: any, encoding?: any, cb?: () => void) => {
-      rowStreamUnpipe(rowStream, userStream);
-      userCanceled = true;
-      if (activeRequestStream) {
-        activeRequestStream.abort();
-      }
-      if (retryTimer) {
-        clearTimeout(retryTimer);
-      }
-      return originalEnd(chunk, encoding, cb);
-    };
-
-    const makeNewRequest = () => {
```

Comment:
The old custom retry logic used to call `makeNewRequest` every time a retry needed to happen. Now retries are done in gax so the indented code that follows this is only called once.